tests/test_frictionless.py

```diff
@@ -0,0 +1,29 @@
+import shutil
+import pathlib
+
+import pytest
+
+from csvw import TableGroup
+from csvw.frictionless import DataPackage
+
+FIXTURES = pathlib.Path(__file__).parent / 'fixtures'
+
+
+@pytest.fixture
+def tmpfixtures(tmpdir):
+    shutil.copytree(str(pathlib.Path(__file__).parent / 'fixtures'), str(tmpdir.join('fixtures')))
+    return pathlib.Path(str(tmpdir)) / 'fixtures'
+
+
+def test_DataPackage(tmpfixtures):
+    dp = DataPackage(tmpfixtures / 'datapackage.json')
+    tg = dp.to_tablegroup()
+    rows = list(tg.tables[0])
+    assert len(rows) == 9
+    assert rows[-1]['Year'] == 2012
+
+    tg.to_file(tmpfixtures / 'metadata.json')
```

Comment:
@brockfanning this is how you'd use this to convert frictionless to CSVW metadata:
```python
dp = DataPackage('PATH/TO/datapackage.json')
tg = dp.to_tablegroup()
tg.to_file(tmpfixtures / 'metadata.json')
```